projects/addon-commerce/components/input-card-grouped/input-card-grouped.template.html

```diff
@@ -8,130 +8,137 @@
     (tuiHoveredChange)="onHovered($event)"
     (tuiActiveZoneChange)="onActiveZoneChange($event)"
 >
-    <tui-svg
-        *ngIf="!!icon"
-        class="card"
-        automation-id="tui-input-card-grouped__icon"
-        [class.card_right]="!hasCleaner"
-        [src]="icon"
-    ></tui-svg>
-    <div class="wrapper">
-        <span *ngIf="hasCardExampleText" class="example-text">
-            {{exampleText}}
-        </span>
-        <label>
-            <span
-                *ngIf="cardNumberTexts$ | async as texts"
-                class="placeholder"
-                [class.placeholder_raised]="placeholderRaised"
-            >
-                {{isMobile ? texts[0] : texts[1]}}
+    <div class="common-wrapper" (scroll)="onScroll($event.currentTarget)">
```

Comment:
Is it possible to do this on `tui-wrapper` without introducing extra element?

Reply:
yes thanks, it works without extra div